catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/MultilingualTest.java

```diff
@@ -0,0 +1,150 @@
+import android.os.SystemClock;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.ui.MainMenuActivity;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Locale;
+
+import static android.support.test.InstrumentationRegistry.getInstrumentation;
+import static android.support.test.espresso.Espresso.onData;
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.assertThat;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+import static org.hamcrest.Matchers.anything;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+
+@RunWith(AndroidJUnit4.class)
+public class MultilingualTest {
+	private int shortSleep = 1000;
+	private static final Locale ARABICLOCALE = new Locale("ar");
+	private static final Locale URDULOCALE = new Locale("ur");
+	private static final Locale FARSILOCALE = new Locale("fa");
+	private static final Locale DEUTSCHLOCALE = Locale.GERMAN;
+	@Rule
+	public ActivityTestRule<MainMenuActivity> mActivityRule = new ActivityTestRule<>(MainMenuActivity.class);
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@Test
+	public void testChangeLanguageToDeutsch() {
+		gotoMultilingualActivity();
+		onData(anything()).inAdapterView(withId(R.id.list_languages)).atPosition(8).perform(click());
+		assertThat("current Locale is not Deutsch", Locale.getDefault().getDisplayLanguage(), is(DEUTSCHLOCALE.getDisplayLanguage()));
+		SystemClock.sleep(shortSleep);
+		String buttonProgramsName = getResourceString(R.string.main_menu_programs);
+		String buttonContinueName = getResourceString(R.string.main_menu_continue);
+		String buttonNewName = getResourceString(R.string.main_menu_new);
+		onView(withId(R.id.main_menu_button_continue)).check(matches(withText(containsString(buttonContinueName))));
+		onView(withId(R.id.main_menu_button_new)).check(matches(withText(buttonNewName)));
+		onView(withId(R.id.main_menu_button_programs)).check(matches(withText(buttonProgramsName)));
+	}
+
+	@Test
+	public void testChangeLanguageToArabic() {
+		gotoMultilingualActivity();
+		onData(anything()).inAdapterView(withId(R.id.list_languages)).atPosition(44).perform(click());
+		assertThat("current Locale is not Arabic", Locale.getDefault().getDisplayLanguage(), is(ARABICLOCALE.getDisplayLanguage()));
+		assertThat("not RTL", true, is(isRTL()));
```

Comment:
theres also 
assertTrue
assertEquals
assert...

please use those, it makes everything a bit more readable. 
eg.
assertTrue(isRTL())

and assertion messages should only be used if the failing asserting would not provide enough information.